scripts/backup_config.sh

```diff
@@ -12,3 +12,4 @@ tar --one-file-system -czf "${backup_filename}" "${HOME}/.config/amplipi/"
 # remove all backups older than 90 days
 echo "removing old backups..."
 find "${backup_dir}" -type f -ctime +90 -delete
+echo "completed ${0}"
```

Comment:
the very last command in a shell script's exit status is considered to be the script's exit status.. and `find` has some funny conditions around returning a non-zero exitstatus with `-delete` :melting_face: 

if we fail to delete backups? don't fail upgrades or consider this a failed process.